spec/classes/mod/log_forensic_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'apache::mod::log_forensic', type: :class do
+  ['Debian 11', 'RedHat 8', 'FreeBSD 9', 'Gentoo'].each do |os|
```

Comment:
"FreeBSD 9" is EOL (same for 10 and 11 and 12 in a few month), I guess this was copy-pasted, maybe we can adjust this (but I do not think any code is FreeBSD-version dependent so I would be surprised if it change anything)?